packages/browser-wallet/src/popup/pages/IdentityIssuance/i18n/en.ts

```diff
@@ -0,0 +1,17 @@
+const t = {
+    title: 'Your Concordium identity',
+    abortExplanation:
+        'The identityissuance was aborted. If you did not abort the process, please try again, or contact support.',
+    successExplanation:
+        'Your request has been submitted with the identity provider. It may take a little while for them to confirm your identity. \n\n Once your identity has been verified, you will be able to open an account with it.',
+    errorExplanation: 'The identityissuance has failed due to: "{{reason}}"',
```

Comment:
```suggestion
    errorExplanation: 'The identity issuance failed due to: "{{reason}}"',
```

Reply:
Also the window that is opened when showing this text has an ugly "X" in the top right corner. Isn't the "Continue" button enough?

Reply:
Well the ugly "X" is a part of the prompt layout, but I guess you also want it removed for signTransaction and signMessage.

Reply:
Yeah, we should just redo that later then.